### DISCUSSION TITLE
[Solved] Sort funktion does not work for me...

### DISCUSSION THREAD

**Discussion by donejones** [Nekmo/telegram-upload#196] [Q&A]
Hello, I just started using tis program. However I already run in to troubles. I can't use --sort and I already installed natsort. I tried --sort berfore and after installing natsort.
```
Input:	telegram-upload --to https://t.me/xyz --directories recursive --sort "/.../.../"

Answer:	Usage: telegram-upload [OPTIONS] [FILES]...
	Try 'telegram-upload --help' for help.

	Error: No such option: --sort Did you mean --to?
```

I want to upload multiple nubered files from a folder in ascending order.
Thank you for your help in advance!

**Comment by Nekmo** [comment]
Hi, what is your telegram-upload version?

**Reply by donejones** [reply]
Thanks solved with updating to 0.6.0. I had 0.5.1. Didn't realise, that my installation did not install the lastest version. Sorry and thank you for your fast reply

**Reply by Nekmo** [reply]
You are welcome:)
